pkgs/kubernetes/kubectl/registry.yaml

```diff
@@ -3,9 +3,8 @@ packages:
     type: http
     repo_owner: kubernetes
     repo_name: kubernetes
-    url: https://storage.googleapis.com/kubernetes-release/release/{{.Version}}/bin/{{.OS}}/{{.Arch}}/kubectl
+    url: https://dl.k8s.io/{{.Version}}/bin/{{.OS}}/{{.Arch}}/kubectl
     format: raw
-    link: https://kubernetes.io/docs/reference/kubectl/overview/
```

Comment:
In this case, we should set `link`, because https://kubernetes.io/docs/reference/kubectl/ is better than the default link https://github.com/kubernetes/kubernetes